NATIVE_SPEND.md

```diff
@@ -55,4 +55,80 @@ JWT need to be signed by server side where private key is secure.
         } catch (TaskFailedException e) {
             e.printStackTrace();
         }
+```
+
+# Add spend offer opportunity to marketplace
+1. Add a NativeOfferCallback to be triggered when the user clicks on your offer in Kin Marketplace.
+```java
+        try {
+            Kin.start(getApplicationContext(), signInData)
+            .addNativeOfferCallback(new Callback<NativeSpendOffer>() {
+                    @Override
+                    public void onResponse(NativeSpendOffer offer) {
+                        // Do checks on the offer
+                      
+                    }
+        
+                    @Override
+                    public void onFailure(Throwable t) {
+        
+                    }
+                });
+        } catch (InitializeException e) {
+            e.printStackTrace();
+        }
+```
+2. Create a [NativeSpendOffer](/kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/marketplace/model/NativeSpendOffer.java) object:
+```java
+    NativeSpendOffer nativeOffer =
+        new NativeSpendOffer("The offerID") // OfferId should be uniqe
+            .title("Offer Title")
+            .description("Offer Description")
+            .amount(int_amount)
+            .image("A URL to offer image");
+```
+3. Add the offer to Kin Marketplace, notice each offer you add will be added at index 0.
+```java
+    try {
+        if (Kin.addNativeOffer(nativeSpendOffer)) {
+            // Native offer added
+        } else {
+            // Native offer already in Kin Marketplace
+        }
+    } catch (TaskFailedException e) {
+        ...
+    }
+```
+
+4. Remove the offer, you can also remove the offer from Kin Marketplace
+```java
+    try {
+        if (Kin.removeNativeOffer(nativeSpendOffer)) {
+             // Native offer removed
+        } else {
+            // Native offer isn't in Kin Marketplace
+        }
+    } catch (TaskFailedException e) {
+        e.printStackTrace();
+    }
+```
+
+# Get Order Confirmation
+if you wish to get the order status of a certain offer, which already completed or<br>
+the user closed the app and you want to get the order status or jwtConfirmation<br>
+just follow the example below:
+```java
+    Kin.getOrderConfirmation("your_offer_id", new Callback<OrderConfirmation>() {
+            @Override
+            public void onResponse(OrderConfirmation orderConfirmation) {
+                if(orderConfirmation.getStatus() == Status.COMPLETED ){
+                   String jwtConfirmation = orderConfirmation.getStatus()
```

Comment:
String jwtConfirmation =orderConfirmation.getJwtConfirmation()

Maybe add OrderConfirmation definition to readme?

Reply:
also perhaps - OrderConfirmationStatus.COMPLETED instead of just Status.COMPLETED

Reply:
@OrenZak  ```String jwtConfirmation = orderConfirmation.getStatus()``` should be ```String jwtConfirmation =orderConfirmation.getJwtConfirmation()``` right?

Reply:
yes